.github/actions/add-issue-to-project/action.yml

```diff
@@ -0,0 +1,49 @@
+---
+name: Add issues to project
+
+description: |
+  This action adds issues to a project if they are not already in the project.
+
+inputs:
+  projectId:
+    description: "Project ID"
+    required: true
+  issueNumber:
+    description: "Issue Number"
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - id: get_project_count
+      uses: octokit/graphql-action@v2.3.2
+      with:
+        # API https://docs.github.com/en/graphql/reference/objects#issue
+        query: |
+          query getProjectCount($owner:String!, $repo:String!, $issue: Int!) {
+            repository(name: $repo, owner: $owner) {
+              issue: issue(number: $issue) {
+                projectsV2 {
+                  totalCount
+                }
+              }
+            }
+          }
+        variables: |
+          owner: "camunda"
+          repo: "camunda"
+          issue: ${{ inputs.issueNumber }}
+      env:
+        GITHUB_TOKEN: ${{ secrets.GH_PROJECT_AUTOMATION_TOKEN }}
```

Comment:
Let's add a new input `githubToken` to pass the secret explicitly from the workflow